easybuild/easyconfigs/f/FastQ_Screen/FastQ_Screen-0.14.1-foss-2019b-Perl-5.30.0.eb

```diff
@@ -0,0 +1,43 @@
+# This file is an EasyBuild reciPY as per https://github.com/easybuilders/easybuild
+
+easyblock = 'Tarball'
+
+name = 'FastQ_Screen'
+version = '0.14.1'
+versionsuffix = '-Perl-%(perlver)s'
```

Comment:
We decided to start dropping the perl suffix a while back.
Though I think we have missed removing it from some configs, so, I'm not entirely sure from when we decided to drop it, but, I think 2019b onwards should have all dropped it.

Reply:
Done.